packages/mobile/lib/auxiliary/drawer/interfaces/drawer.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { DrawerId } from '../enums'
+import { DrawerProps } from '../types'
+
+export interface IDrawer {
```

Comment:
I think renaming this to `IDrawerState` might help it be more consistent with the store.